spec/lib/sidekiq/throttled/patches/super_fetch_spec.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+require "sidekiq/throttled/patches/super_fetch"
+begin
+  require "sidekiq/pro/super_fetch"
+rescue LoadError
+  # Sidekiq Pro is not available
+end
```

Comment:
I don't think we need this:

```suggestion
```

After `require "sidekiq/throttled/patches/super_fetch"` we can safely assume we did that already

Reply:
You're right. Specs still pass without it 💪🏽